recipe/meta.yaml

```diff
@@ -45,7 +35,7 @@ test:
 about:
   home: http://www.algorithm.uni-bayreuth.de/en/research/SYMMETRICA/index.html
   license: ISC
-  license_file: {{ RECIPE_DIR }}/LICENSE
+  license_file: LICENSE
   summary: |
     Symmetrica is a collection of routines to handle several topics including
     ordinary representation theory of the symmetric group and related groups
```

Comment:
@alexjbest could you add the two of us as maintainers here before this gets merged?

Reply:
@saraedum Done!